app.py

```diff
@@ -158,7 +158,9 @@ def process_error(error):
     if error.find("Your code has been rated at") > -1:
         return None
 
-    line_num = error.split(":")[1]
+    # Linux would return error message with line number on position [1] 
+    # and windows place line number on position [2]
+    line_num = error.split(":")[1] if error.split(":")[1].isnumeric() else error.split(":")[2] 
```

Comment:
Can we use the `os` library to detect if the user's operating system is Windows or Linux instead of detecting it via Pylint. There could be some edge cases where this is triggered and the operating system is not detected correctly. 

Here's a link to get you started: https://stackoverflow.com/a/1325587/4698963